Makefile

```diff
@@ -100,12 +100,10 @@ generate: ## Run go generate
 
 .PHONY: generate-crds
 generate-crds: ## Generate CRDs and Go types using kubebuilder
-	go run sigs.k8s.io/controller-tools/cmd/controller-gen crd paths=./apis/... output:crd:dir=deploy/helm-chart/crds
```

Comment:
We still need to generate the CRDs from the Go code and put them in the new location.

Reply:
We do, did I miss something?

Reply:
Oh my bad, I didn't see the change to the line below